aws/src/main/java/org/apache/iceberg/aws/AwsClientFactory.java

```diff
@@ -56,6 +57,12 @@
    */
   DynamoDbClient dynamo();
 
+  /**
+   * Create a Amazon S3Control client
+   * @return s3Control client
+   */
+  S3ControlClient s3Control();
```

Comment:
I think we no longer need this in the client factory? It is for test only and we already have a method there to create s3 control client.